src/components/AuthContainer.tsx

```diff
@@ -1,14 +1,18 @@
 import React, { useContext, useEffect } from 'react';
-import { Button, SafeAreaView, Text } from 'react-native';
+import { Button, SafeAreaView } from 'react-native';
 import { withAuthenticator, AmplifyTheme } from 'aws-amplify-react-native';
 import { Auth, I18n } from 'aws-amplify';
 import { Translations } from '@aws-amplify/ui-components';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import Onboarding from './Onboarding/Onboarding';
 import AuthenticatorTheme from '../styles/AuthenticatorTheme';
 import { AppContext } from '../contexts/AppContext';
+import NavFlow from './NavContainer';
 
 Auth.configure({ mandatorySignIn: true });
 
+const Tab = createBottomTabNavigator();
```

Comment:
I don't see this used anywhere below, so probably remove this and the associated import statement?

Reply:
@thaigillespie Yep. appears to have been used as debugging fluff in the form of: `<Text>Use React Navigation here</Text>`. Looks like we can remove it.

Reply:
Good catch! I'll take that out. @unsoupervised was spot on.